packages/Webkul/Shop/src/Resources/views/customers/account/wishlist/index.blade.php

```diff
@@ -0,0 +1,39 @@
+<x-shop::layouts.account>
+    <div class="max-lg:hidden">
+        <div class="flex gap-x-[4px] items-center mb-[10px]">
+            <p class="flex items-center gap-x-[4px] text-[#7D7D7D] text-[16px] after:content-['/']">
+                @lang('shop::app.customers.account.wishlist.profile')
+            </p>
+
+            <p class="flex items-center gap-x-[4px] text-[#7D7D7D] text-[16px] after:content-['/'] after:last:hidden">
+                @lang('shop::app.customers.account.wishlist.title')
+            </p>
+        </div>
+
+        <h2 class="text-[26px] font-medium">
+            @lang('shop::app.customers.account.wishlist.page-title')
+        </h2>
+
+        <!-- wishlist Products -->
```

Comment:
single line comment
{{-- wishlist producs --}}

multiline comment
{{--
    Wishlist Products:
    This is a wishlist product.
--}}

Reply:
Done